src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,285 @@
+
+        System.out.println("I'll delete this:");
+        task.printTask();
+        System.out.println("Now you have " + tasks.size() + " tasks in the list!");
+
+    }
+
+
+    public static void addTodo(String line, Boolean fromFile) {
+        tasks.add(new Todo(line));
+
+        if (!fromFile) {
+            writeToFile("duke.txt", tasks.get(tasks.size()-1).toString());
+        }
+    }
+
+    public static void addDeadline(String line, Boolean fromFile) {
+        String[] descriptionBy = line.split(" /by ");
+       // list[size++] = new Deadline(descriptionBy[0], descriptionBy[1]);
+        tasks.add(new Deadline(descriptionBy[0], descriptionBy[1]));
+
+        if (!fromFile) {
+            writeToFile("duke.txt", tasks.get(tasks.size()-1).toString());
+        }
+
+    }
+
+    public static void addEvent(String line, Boolean fromFile) {
+        String[] descriptionAt = line.split(" /at ");
+
+      //  list[size++] = new Event(descriptionAt[0], descriptionAt[1]);
+        tasks.add(new Event(descriptionAt[0], descriptionAt[1]));
+        if (!fromFile) {
+            writeToFile("duke.txt", tasks.get(tasks.size()-1).toString());
+        }
+    }
+
+
+    public static void printExitMessage() {
+
+        System.out.println("    ____________________________________________________________");
+        System.out.println("    Bye. Hope to see you again soon!");
+        System.out.println("    ____________________________________________________________");
+
+    }
+
+
+    public static void createFile() throws IOException {
+        File file = new File("duke.txt");
+
+        if (file.exists() == false) {
+            file.createNewFile();
+        }
+        Scanner read = new Scanner(file);
+        while (read.hasNext()) {
+            loadFileTask(read.nextLine());
+        }
+        read.close();
+    }
+
+    public static void loadFileTask(String line) {
+        String[] taskDetails = line.split(" \\| ");
+
+        switch (taskDetails[0]) {
+        case "[T]":
+            addTodo(taskDetails[2], true);
+            break;
+        case "[D]":
+            addDeadline(taskDetails[2] + " /by " + taskDetails[3], true);
+            break;
+        case "[E]":
+            addEvent(taskDetails[2] + " /at " + taskDetails[3], true);
+            break;
+        default:
+            // Happens if there is nothing in the list (or any garbage text)
+            return;
+        }
+
+        // Mark as done if the task is already done
+        if (taskDetails[1].equals("1")) {
+            tasks.get(tasks.size()-1).isDone = true;
+        }
+    }
+
+    public static void writeToFile(String filePath, String line) {
+        try {
+            FileWriter writer = new FileWriter(filePath, true);
+            writer.write(System.lineSeparator() + line);
+            writer.close();
+        } catch (IOException e) {
+            System.out.println("Something went wrong: " + e.getMessage());
+        }
+    }
+
+    /*
```

Comment:
This javadoc has a wrong format. You can see how block of comments are structured or how a line of comment is written (see screenshot below).

![Screenshot 2020-09-15 at 1 07 55 AM](https://user-images.githubusercontent.com/59989652/93116489-e5b9d800-f6ef-11ea-87a9-2395f60a3ca5.png)